frontend/www/js/omegaup/components/arena/ContestListv2.vue

```diff
@@ -339,6 +343,7 @@ export enum ContestOrder {
   },
 })
 export default class ArenaContestList extends Vue {
+  @Prop() countContests!: { [key: string]: number };
```

Comment:
To avoid modify the existing tests, it should be a good idea set null as default value:
```suggestion
  @Prop({ default: null }) countContests!: { [key: string]: number } | null;
```